ptf/config/upf.jsonc

```diff
@@ -155,7 +155,12 @@
     "cpiface": {
         "peers": ["148.162.12.214"],
         // [Optional] Below parameters
-        "dnn": "internet",
+        "dnn_list": [
+            {
+                "dnn": "internet",
+                "ue_ip_pool": "10.250.0.0/16"
```

Comment:
Same comment as in the other jsonc file. Should not the `ue_ip_pool` be removed from below (new line 168)